internal/vm/dir.go

```diff
@@ -183,7 +185,7 @@ func relPathTo(absPath string) (string, error) {
 
 	relPath, err := filepath.Rel(cwd, absPath)
 	if err != nil {
-		return "", errors.Wrapf(err, "failed to get relative path from %q to %q", cwd, absPath)
+		return absPath, nil
```

Comment:
If anyone feels ill about this change, another option is to return either a typed error or a known error. That way we can check against the error type and just use the absolute path instead. However, we'd do that literally for all relative methods, which is why I decided to just return the absolute path here

Reply:
We wanted to return relative paths to workaround #264. Wouldn't we hit the issue again by returning absolute paths?